.github/ISSUE_TEMPLATE/quick-wins.md

```diff
@@ -0,0 +1,14 @@
+---
+name: Quick Win
+about: Quick Win tasks to be taken by the community.
+title: ''
+labels: good first issue
+assignees: ''
+
+---
+
+Issues should only be opened by the repository maintainers. 
+
+Bugs should be reported to customer support and feature requests should be posted to AHA. 
+
+Please do not open this issue.
```

Comment:
Not clear

Reply:
not sure if you're serious, how can we be more clear than that?

Reply:
I was actually serious lol. Why not put this at the beginning and say something like: Please do not open this issue - this is for internal usage only.

Reply:
this is pretty much what it says, would you like to rephrase it ?